client/app/routes/collection/add.js

```diff
@@ -1,4 +1,124 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+
+    model() {
+        return {
+            submission_form_name: 'Preprints Submission Form',
+            sections: [
+                'upload',
+                'disciplines',
+                'basic info',
+                'authors',
+                'submit'
+            ],
+            initial_state: {
+                upload_section: {
+                    state: ['unsaved', 'editing'],
+                    value: undefined
```

Comment:
could be something like `document.getElementById('#upload_section')`, so entire sections can be toggled hidden or animated.